website/pages/en/billing.mdx

```diff
@@ -194,3 +194,16 @@ Of course, both new and existing users can reach out to Edge & Node's BD team fo
 ### Can I withdrawal GRT from my billing balance?
 
 Yes, you can always withdrawal GRT that has not already been used for queries from your billing balance. The billing contract is only designed to bridge GRT from Ethereum mainnet to the Arbitrum network. If you'd like to transfer your GRT from Arbitrum back to Ethereum mainnet, you'll need to use the [Arbitrum Bridge](https://bridge.arbitrum.io/?l2ChainId=42161).
+
+### What happens when GRT is depleted? Will I get a warning?
+
+Users will receive an email when GRT is close to depletion. You will receive several notifications before billing runs out.
```

Comment:
This was the suggested language I received, but happy to change it.

Reply:
I just checked the question above this one in case it adds some context that I was missing, and it does mention "my billing balance" there, which I think is much clearer. By the way, I can't make a suggestion inline because you haven't changed those lines, but there is a typo: "Can I withdrawal GRT" should be "Can I _withdraw_ GRT" and the same error is present in on the paragraph that follows: "Yes, you can always _withdraw_". If you could fix those it would be awesome. 🙏

Reply:
I did not edit the rest of this page at all, so thank you so much for bringing this to my attention. I will fix right away.